src/main/java/com/cx/restclient/httpClient/CxHttpClient.java

```diff
@@ -600,8 +600,13 @@ public <T> T getRequest(String rootURL, String relPath, String acceptHeader, Str
     //POST REQUEST
     public <T> T postRequest(String relPath, String contentType, HttpEntity entity, Class<T> responseType, int expectStatus, String failedMsg) throws IOException {
         HttpPost post = new HttpPost(rootUri + relPath);
-        return request(post, contentType, entity, responseType, expectStatus, failedMsg, false, true);
-    }
+		if ("sast/scanWithSettings".equals(relPath)) {
```

Comment:
This is no correct,   CxHTTPClient is a generic HTTPClient.  We cannot have API endpoints in the logic here. Instead, logic can be on the basis of Content-Type